plugins/catalog-backend/src/service/AuthorizedEntitiesCatalog.ts

```diff
@@ -264,6 +266,23 @@ export class AuthorizedEntitiesCatalog implements EntitiesCatalog {
     };
   }
 
+  async unprocessed(
```

Comment:
I'm wondering if we should make this whole extension a module that you can install into the catalog instead of building it directly into the catalog backend plugin. It would make things simpler in the catalog-backend and encourage people to start breaking out these extensions into modules instead.

Reply:
Yep I think module is the best approach here idd. A plain `createBackendModule` for the new system, and for the old one a separate `createUnprocessedEntitiesRouter` that you install alongside the main router in the catalog backend.

Reply:
My first idea was to add an `addAdditionalRoutes` function to the CatalogBuilder which I think would be cleaner, but the problem with that was that the EntitiesCatalog gets built in the Builder. Since it's needed in the route handlers I'm not sure how to solve the dependency injection.